arrows-es6/index.html

```diff
@@ -0,0 +1,85 @@
+---
+feature_name: ES6 Arrow functions
+chrome_version: 45
+feature_id: 5047308127305728
+additional_head_content: <meta name="optional" value="optional">
```

Comment:
This isn't needed—it's just showing off how to include extra tags in the `<head>`.